hail/src/main/scala/is/hail/linalg/BlockMatrix.scala

```diff
@@ -1213,7 +1213,7 @@ class BlockMatrix(val blocks: RDD[((Int, Int), BDM[Double])],
         }
     }
 
-    new Table(hc, entriesRDD, rvRowType, Array("i", "j"))
+    new Table(hc, entriesRDD, rvRowType, Array[String]())
```

Comment:
this is technically a user-facing change, right? That's OK since this is experimental, but I'll remember to put it in the change log.

Reply:
Fixed by adding an optional `keyed=True` parameter in python which calls `key_by` if necessary.